tests/integration/ceph_base_deploy_test.go

```diff
@@ -40,12 +40,12 @@ const (
 	// These versions are for running a minimal test suite for more efficient tests across different versions of K8s
 	// instead of running all suites on all versions
 	// To run on multiple versions, add a comma separate list such as 1.16.0,1.17.0
-	flexDriverMinimalTestVersion      = "1.15.0"
-	cephMasterSuiteMinimalTestVersion = "1.16.0"
-	multiClusterMinimalTestVersion    = "1.16.0"
+	flexDriverMinimalTestVersion      = "1.11.0"
+	cephMasterSuiteMinimalTestVersion = "1.12.0"
+	multiClusterMinimalTestVersion    = "1.15.0"
 	helmMinimalTestVersion            = "1.17.0"
 	upgradeMinimalTestVersion         = "1.18.0"
-	smokeSuiteMinimalTestVersion      = "1.19.0"
+	smokeSuiteMinimalTestVersion      = "1.13.0,1.19.0"
```

Comment:
should this go back to 1.11 also?

Reply:
This is just the list of versions where the test suites will run for PRs where we don't run the full matrix, so it's not needed on 1.11.